Doc/library/collections.rst

```diff
@@ -1035,8 +1036,8 @@ anywhere a regular dictionary is used.
 
 .. versionchanged:: 3.6
    With the acceptance of :pep:`468`, order is retained for keyword arguments
-   passed to the :class:`OrderedDict` constructor and its :meth:`update`
-   method.
+   passed to the :class:`OrderedDict` constructor and its
+   :meth:`update` method.
```

Comment:
Nothing. 

I originally changed it to `dict.update` but then I wasn't sure if that was what was intended and reverted it again. Just forgot to revert all of the change.